src/rcx/src/extSpef.cpp

```diff
@@ -87,6 +87,13 @@ extSpef::~extSpef()
   delete _nodeCoordParser;
   free(_msgBuf1);
   free(_msgBuf2);
+
+  for (uint ii = 0; ii < 1024; ii++) {
```

Comment:
It would be better to use `_hcnrc->getSize()` than `1024` in case it ever gets resized.